qiskit/scheduler/__init__.py

```diff
@@ -19,11 +19,8 @@
 
 A circuit scheduler compiles a circuit program to a pulse program.
 
-.. autosummary::
-   :toctree: ../stubs/
-
-   schedule_circuit
-   ScheduleConfig
+.. autofunction:: qiskit.scheduler.schedule_circuit.schedule_circuit
```

Comment:
This seems like it's probably not documenting exactly the same object, though the function is more correctly what should be documented.  We probably need to make sure its complete import path is shown here, like mentioned with the `.. currentmodule` stuff elsewhere.

Reply:
Yeah, this one took me a while to figure out because Sphinx was complaining cryptically that `schedule_circuit` was not a callable - indeed, I was using `autofunction` on the module. We made you first go to a module page with only one entry, the duplicate-named function.